easybuild/easyconfigs/h/HTSlib/HTSlib-1.9-foss-2019a.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# 1.4 modified by:
+# Adam Huffman, Jonas Demeulemeester
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+name = 'HTSlib'
+version = '1.9'
+
+homepage = "http://www.htslib.org/"
+description = """A C library for reading/writing high-throughput sequencing data.
+ This package includes the utilities bgzip and tabix"""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```

Comment:
this can probably be moved down to the `GCC` toolchain?

Reply:
Good idea. I moved it too `GCCcore though`, as HTSlib could still be used as a dependency in the `intel` toolchain.

Reply:
Moved it to `GCC`